src/rhash/hash.c

```diff
@@ -750,7 +750,7 @@ static int rc_hash_jaguar_cd(char hash[33], const char* path)
   char message[128];
   void* track_handle;
   md5_state_t md5;
-  int byteswapped;
+  int byteswapped = -1;
```

Comment:
Recommend setting to 0. This is effectively a boolean, so -1 is invalid (and would evaluate as true).